django/urls/converters.py

```diff
@@ -53,6 +56,15 @@ class PathConverter(StringConverter):
 
 
 def register_converter(converter, type_name):
+    if type_name in REGISTERED_CONVERTERS or type_name in DEFAULT_CONVERTERS:
+        # Removed in Django60Warning: when the deprecation end, replace with
+        # raise ValueError(f"converter {type_name} already exists")
```

Comment:
```suggestion
        # raise ValueError(f"Converter {type_name} already exists.")
```
Looking other exceptions we have a convention of correct grammar including full stops.

Reply:
Thanks!